app/models/concerns/hyrax/file_set/indexing.rb

```diff
@@ -6,7 +6,7 @@ module Indexing
 
       included do
         # the default indexing service
-        self.indexer = Hyrax::FileSetIndexer
+        self.indexer = Hyrax::ActiveFedoraFileSetIndexer
```

Comment:
Completes the refactor moving the AF indexing to `Hyrax::ActiveFedoraFileSetIndexer` _(for AF objects)_ out of `Hyrax::FileSetIndexer` _(now being used for Valkyrie resource `Hyrax::FileSet`)_.